### PR TITLE
Initial content of the EyeDropper API

### PR DESCRIPTION
**Pull request by captainbrosset**
#### Summary
This adds some initial documentation content for the EyeDropper API.

The suggestion to add docs for this API was made in issue #9955.

#### Motivation
The API is implemented in Chromium-based browsers, and can therefore already be used (Chrome 95+ or Edge 95+ for example).

#### Metadata
- [x] Adds a new document

#### Note

This is my first time creating an entirely new API documentation on MDN, so let me know if I forgot some pages or messed up some cross-linking.

Also, I'm not sure I understand how to add the spec section on all of the new pages.

Finally, there isn't any BCD data yet to go with this. I suppose this needs to be done separately in the BCD repository.

### CODE REVIEW

**Comment by wbamberg** [comment]
Thanks @captainbrosset ! I'll take a look.

> Also, I'm not sure I understand how to add the spec section on all of the new pages.

The BCD should include a `spec_url` property that points to, well a URL for the bit of the spec for that feature: https://github.com/mdn/browser-compat-data/blob/main/schemas/compat-data-schema.md#the-__compat-object.

Then the page needs to include:
* the "browser-compat" front matter property, whose value is the BCD query
* a line like {{Specifications}} where you want the spec table to appear
* a line like {{Compat}} where you want the BCD table to appear

For example:

* BCD for `Request.url`: https://github.com/mdn/browser-compat-data/blob/main/api/Request.json#L1296
* front matter property: https://github.com/mdn/content/blob/f00aff507b8e39ccef4d04e5013b2e9ceacc58ae/files/en-us/web/api/request/url/index.md?plain=1#L12
* {{Specifications}} and {{Compat}}: https://github.com/mdn/content/blob/f00aff507b8e39ccef4d04e5013b2e9ceacc58ae/files/en-us/web/api/request/url/index.md?plain=1#L40-L46

> Finally, there isn't any BCD data yet to go with this. I suppose this needs to be done separately in the BCD repository.

Yes.

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for this @captainbrosset !

**Review by wbamberg — APPROVED** [review]
👍 looks great, that you @captainbrosset !

File: files/en-us/web/api/eyedropper/index.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    resultElement.style.backgroundColor = result.sRGBHex;
+  }).catch(e => {
+    resultElement.textContent = e;
+  });
+});
+```
+
+#### Result
+
+{{EmbedLiveSample("Opening the eyedropper tool and sampling a color")}}
```
[the remark this answers is not in the document]
Comment (escattone) [reply files/en-us/web/api/eyedropper/index.md:65]: @wbamberg Yes, that does work! 🚀

**Pull request merged**